src/app/views/marketplace/marketplace-compare.html

```diff
@@ -0,0 +1,73 @@
+<div class="dashboard-marketplace-compare">
+  <mno-loader ng-show="vm.isLoading" class="top-buffer-4"></mno-loader>
+  <!-- Header -->
+  <div class="row show-header top-buffer-4">
+    <div class="col-md-12">
+      <a ui-sref="home.marketplace()" class="text-muted">
+        <i class="fa fa-chevron-left"></i> {{ 'mno_enterprise.templates.dashboard.marketplace.show.back' | translate }}
+      </a>
+    </div>
+  </div>
+  <div class="marketplace-compare-wrap" ng-show="!vm.isLoading">
+    <div class="row">
+      <div class="col-md-12">
+        <h1>Comparison</h1>
+        <!-- <h2>{{vm.greeting.message}}</h2> -->
+        <div class="marketplace-compare-info">
+          <table class="marketplace-compare-table table table-bordered">
+            <tr>
+              <th></th>
+              <td ng-repeat="app in vm.apps | limitTo:4"><a ui-sref="home.marketplace.app({appId: app.id})"><img class="media-object" ng-src="{{app.logo}}" ng-alt="{{app.name}}"></a></td>
+            </tr>
+            <tr>
+              <th>Application Provider</th>
+              <td ng-repeat="app in vm.apps | limitTo:4">{{app.name}}</td>
+            </tr>
+            <tr>
+              <th>Category</th>
+              <td ng-repeat="app in vm.apps | limitTo:4">{{app.categories[2]}}</td>
+            </tr>
+            <tr>
+              <th>Key benefits</th>
+              <td ng-repeat="app in vm.apps | limitTo:4">{{app.key_benefits}}</td>
```

Comment:
Looks like `app.key_benefits` is an array, might want to consider not using `.join` to prevent the array being rendered... unless that was specified?